### ISSUE TITLE
Feature request: read from onboard DMP

### ISSUE THREAD

**Issue by John-Trager** [adafruit/Adafruit_CircuitPython_MPU6050#18]
The 6050 MPU has a Digital Motion Processor (DMP) chip onboard that runs motion algorithms and sensor fusion to create more accurate state data. If this could be read and accessed using this package that would be great!

For reference, I believe this library https://github.com/jrowberg/i2cdevlib/tree/master/Arduino/MPU6050 Have figure it out.

**Comment by jposada202020** [comment]
Thanks, there are not plans that I know of to add this feature at the moment.
However, we consider any community PR contribution. Let us know if you could provide one. Thanks.

**Comment by SquirtleSquadLeader** [comment]
@ladyada  I am currently using this sensor on a project.  I would like to contribute to this particular sensors CP module.    

Are there any submission guidelines for Adafruit libraries?  Or just submit working code and see if it gets accepted???  I never participated in any opensource stuff before so I am a little bit of a rookie in this department.  I appreciate any guidance!

**Comment by ladyada** [comment]
yep here's a guide - https://learn.adafruit.com/contribute-to-circuitpython-with-git-and-github - if you follow those steps you will make a PR we can review for inclusion :)

**Comment by SquirtleSquadLeader** [comment]
@ladyada Awesome.  Thank you.  Time to get to work.
